core/common/src/main/java/alluxio/underfs/UnderFileSystemFactoryRegistry.java

```diff
@@ -153,28 +163,50 @@ public static UnderFileSystemFactory find(String path) {
   }
 
   /**
+   * Finds all {@link UnderFileSystemFactory} from the lib directory and caches.
+   */
+  private static void loadLibs() {
+    List<File> files = new ArrayList<>();
+    try (DirectoryStream<Path> stream = Files.newDirectoryStream(
+        Paths.get(Configuration.get(PropertyKey.HOME), "lib"), "alluxio-underfs-*.jar")) {
```

Comment:
seems like the lib directory might be used by non-underfs libs as well. can we define the location as a constant (not in ufs module) or a config parameter if it makes sense. well maybe not a config as i guess the build script will write to a constant path.

Reply:
there should also be a log message stating which dir is being searched for jars.

Reply:
done and done